traits/trait_types.py

```diff
@@ -3336,6 +3338,124 @@ def as_ctrait(self):
         """
         return self.trait_maker.as_ctrait()
 
+# -------------------------------------------------------------------------------
+#  'None' trait:
+# -------------------------------------------------------------------------------
+
+
+class NoneTrait(TraitType):
```

Comment:
Probably should have a `default_value` of `None` and an `__init__` which only accepts metadata so you can't pass in a non-`None` value.

Reply:
Got it.